README.md

```diff
@@ -1,21 +1,36 @@
 # Welcome to DuckDuckHack!
 
-![](http://docs.duckduckhack.com/assets/hack search engine.png)
+We're a community dedicated to improving search results with open source, "Instant Answers", that solve searches in few or zero clicks.
 
-We're a community dedicated to improving search results with open source, "Instant Answers." Instant Answers use the Web's best APIs and data to solve searches in few or zero clicks. Together and openly, we can create the best search engine for every type of search.
 
-**The DuckDuckHack community and DuckDuckGo staff are working together on a mission to create the best search engine for programmers. To fully focus there, we are only accepting Pull Requests and Issues related to the [Programming Mission](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53).**
+### The Programming Mission
+The current focus of DuckDuckHack is to create the best search engine for programmers. For now, we are **only accepting Pull Requests and Issues related to the Programming Mission**.
 
-[**Join the Programming Mission!**](https://forum.duckduckhack.com/t/duckduckhack-programming-mission-overview/53)  
-*Your help and expertise can have a huge impact on improving the search experience for developers and DuckDuckGo users around the world!*
 
-More Resources:   
-*[Full documentation](http://docs.duckduckhack.com)*  
-*[Instant Answers in Production](https://duck.co/ia)*  
-*[Instant Answers in Beta](https://beta.duckduckgo.com/?q=test&t=hc&ia=iatesting&iax=1)*  
+## Get Started
```

Comment:
While we should have a "Getting started" or "How to contribute" section, we shouldn't duplicate content that's already elsewhere IMO. If we have a single "How to contribute" guide on the forum or in CONTRIBUTING.md, we should simply link there. This would reduce the likelihood of the various copies getting out-of-sync over time.